컴파일 할 때 나오는 오류 해결하기

```
> Task :compileJava
/home/runner/work/towork-backend/towork-backend/src/main/java/com/backend/towork/member/domain/entity/Member.java:54: warning: @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
    List<Workspace> workspaces = new ArrayList<>();
                    ^
/home/runner/work/towork-backend/towork-backend/src/main/java/com/backend/towork/member/domain/entity/Member.java:57: warning: @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
    List<Participant> participants = new ArrayList<>();
```

인스턴스를 생성했는데 builder가 적용되서 그런듯 하다.

--- a/src/main/java/com/backend/towork/member/domain/entity/Member.java
+++ b/src/main/java/com/backend/towork/member/domain/entity/Member.java
@@ -50,9 +50,11 @@ public class Member {
     @CreatedDate
     private LocalDateTime createDateTime;
 
+    @Builder.Default
     @OneToMany(mappedBy = "owner")
     List<Workspace> workspaces = new ArrayList<>();
 
+    @Builder.Default
     @OneToMany(mappedBy = "member")
     List<Participant> participants = new ArrayList<>();
 

